src/chat-panel/components/chatPanel.css

```diff
@@ -51,6 +51,18 @@
   white-space: pre-wrap;
   margin-top: 1%;
   border-radius: 5px;
+  position: relative;
+}
+
+.messageDivActive:hover .messageEdit {
+  opacity: 1;
+}
+
+.messageEdit {
+  opacity: 0;
+  position: absolute;
+  transform: translate(-50%, -50%);
```

Comment:
You're missing some positioning properties:
```css
top: 0;
left: 0;
height: 100%;
```

Can also use `bottom: 0;` instead of `height`

Reply:
Fixed this now